2021-07-22 18:00:36,677 p=1 u=root |  ansible 2.8.4
  config file = /ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.7.5 (default, Oct 17 2019, 12:25:15) [GCC 8.3.0]
2021-07-22 18:00:36,677 p=1 u=root |  Using /ansible/ansible.cfg as config file
2021-07-22 18:00:36,727 p=1 u=root |  host_list declined parsing /ansible/hosts as it did not pass it's verify_file() method
2021-07-22 18:00:36,728 p=1 u=root |  script declined parsing /ansible/hosts as it did not pass it's verify_file() method
2021-07-22 18:00:36,728 p=1 u=root |  auto declined parsing /ansible/hosts as it did not pass it's verify_file() method
2021-07-22 18:00:36,733 p=1 u=root |  Parsed /ansible/hosts inventory source with ini plugin
2021-07-22 18:00:37,640 p=1 u=root |  META: ran handlers
2021-07-22 18:00:37,729 p=1 u=root |  <ec2-52-67-136-52.sa-east-1.compute.amazonaws.com> ESTABLISH PARAMIKO SSH CONNECTION FOR USER: ubuntu on PORT 22 TO ec2-52-67-136-52.sa-east-1.compute.amazonaws.com
2021-07-22 18:00:38,000 paramiko.transport starting thread (client mode): 0x9514e610
2021-07-22 18:00:38,001 paramiko.transport Local version/idstring: SSH-2.0-paramiko_2.4.2
2021-07-22 18:00:38,052 paramiko.transport Remote version/idstring: SSH-2.0-OpenSSH_7.6p1 Ubuntu-4ubuntu0.3
2021-07-22 18:00:38,052 paramiko.transport Connected (version 2.0, client OpenSSH_7.6p1)
2021-07-22 18:00:38,074 paramiko.transport kex algos:['curve25519-sha256', 'curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group16-sha512', 'diffie-hellman-group18-sha512', 'diffie-hellman-group14-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2021-07-22 18:00:38,075 paramiko.transport Kex agreed: ecdh-sha2-nistp256
2021-07-22 18:00:38,075 paramiko.transport HostKey agreed: ssh-ed25519
2021-07-22 18:00:38,075 paramiko.transport Cipher agreed: aes128-ctr
2021-07-22 18:00:38,076 paramiko.transport MAC agreed: hmac-sha2-256
2021-07-22 18:00:38,076 paramiko.transport Compression agreed: none
2021-07-22 18:00:38,109 paramiko.transport kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2021-07-22 18:00:38,110 paramiko.transport Switch to new keys ...
2021-07-22 18:00:38,119 p=1 u=root |  ec2-52-67-136-52.sa-east-1.compute.amazonaws.com | UNREACHABLE! => {
    "changed": false,
    "msg": "[Errno 2] No such file or directory: 'Web_SP.pem'",
    "unreachable": true
}
2021-07-22 18:00:40,552 p=1 u=root |  ERROR! the playbook: docker_ubuntu.yml could not be found
2021-07-23 21:17:23,270 p=1 u=root |  ansible 2.8.4
  config file = /ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.7.5 (default, Oct 17 2019, 12:25:15) [GCC 8.3.0]
2021-07-23 21:17:23,270 p=1 u=root |  Using /ansible/ansible.cfg as config file
2021-07-23 21:17:23,314 p=1 u=root |  host_list declined parsing /ansible/hosts as it did not pass it's verify_file() method
2021-07-23 21:17:23,315 p=1 u=root |  script declined parsing /ansible/hosts as it did not pass it's verify_file() method
2021-07-23 21:17:23,315 p=1 u=root |  auto declined parsing /ansible/hosts as it did not pass it's verify_file() method
2021-07-23 21:17:23,320 p=1 u=root |  Parsed /ansible/hosts inventory source with ini plugin
2021-07-23 21:17:24,090 p=1 u=root |  META: ran handlers
2021-07-23 21:17:24,181 p=1 u=root |  <ec2-15-228-54-89.sa-east-1.compute.amazonaws.com> ESTABLISH PARAMIKO SSH CONNECTION FOR USER: ubuntu on PORT 22 TO ec2-15-228-54-89.sa-east-1.compute.amazonaws.com
2021-07-23 21:17:24,364 paramiko.transport starting thread (client mode): 0xc2181750
2021-07-23 21:17:24,365 paramiko.transport Local version/idstring: SSH-2.0-paramiko_2.4.2
2021-07-23 21:17:24,399 paramiko.transport Remote version/idstring: SSH-2.0-OpenSSH_7.6p1 Ubuntu-4ubuntu0.3
2021-07-23 21:17:24,400 paramiko.transport Connected (version 2.0, client OpenSSH_7.6p1)
2021-07-23 21:17:24,423 paramiko.transport kex algos:['curve25519-sha256', 'curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group16-sha512', 'diffie-hellman-group18-sha512', 'diffie-hellman-group14-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2021-07-23 21:17:24,423 paramiko.transport Kex agreed: ecdh-sha2-nistp256
2021-07-23 21:17:24,424 paramiko.transport HostKey agreed: ssh-ed25519
2021-07-23 21:17:24,424 paramiko.transport Cipher agreed: aes128-ctr
2021-07-23 21:17:24,424 paramiko.transport MAC agreed: hmac-sha2-256
2021-07-23 21:17:24,425 paramiko.transport Compression agreed: none
2021-07-23 21:17:24,460 paramiko.transport kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2021-07-23 21:17:24,462 paramiko.transport Switch to new keys ...
2021-07-23 21:17:24,472 p=1 u=root |  ec2-15-228-54-89.sa-east-1.compute.amazonaws.com | UNREACHABLE! => {
    "changed": false,
    "msg": "[Errno 2] No such file or directory: 'Web_SP.pem'",
    "unreachable": true
}
2021-07-23 21:17:26,955 p=1 u=root |  ERROR! the playbook: docker_ubuntu.yml could not be found
2021-07-23 21:21:44,498 p=1 u=root |  ansible 2.8.4
  config file = /ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.7.5 (default, Oct 17 2019, 12:25:15) [GCC 8.3.0]
2021-07-23 21:21:44,498 p=1 u=root |  Using /ansible/ansible.cfg as config file
2021-07-23 21:21:44,543 p=1 u=root |  host_list declined parsing /ansible/hosts as it did not pass it's verify_file() method
2021-07-23 21:21:44,543 p=1 u=root |  script declined parsing /ansible/hosts as it did not pass it's verify_file() method
2021-07-23 21:21:44,543 p=1 u=root |  auto declined parsing /ansible/hosts as it did not pass it's verify_file() method
2021-07-23 21:21:44,548 p=1 u=root |  Parsed /ansible/hosts inventory source with ini plugin
2021-07-23 21:21:45,234 p=1 u=root |  META: ran handlers
2021-07-23 21:21:45,317 p=1 u=root |  <ec2-18-231-152-229.sa-east-1.compute.amazonaws.com> ESTABLISH PARAMIKO SSH CONNECTION FOR USER: ubuntu on PORT 22 TO ec2-18-231-152-229.sa-east-1.compute.amazonaws.com
2021-07-23 21:21:45,499 paramiko.transport starting thread (client mode): 0x3ff8490
2021-07-23 21:21:45,500 paramiko.transport Local version/idstring: SSH-2.0-paramiko_2.4.2
2021-07-23 21:21:45,538 paramiko.transport Remote version/idstring: SSH-2.0-OpenSSH_7.6p1 Ubuntu-4ubuntu0.3
2021-07-23 21:21:45,539 paramiko.transport Connected (version 2.0, client OpenSSH_7.6p1)
2021-07-23 21:21:45,560 paramiko.transport kex algos:['curve25519-sha256', 'curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group16-sha512', 'diffie-hellman-group18-sha512', 'diffie-hellman-group14-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2021-07-23 21:21:45,560 paramiko.transport Kex agreed: ecdh-sha2-nistp256
2021-07-23 21:21:45,561 paramiko.transport HostKey agreed: ssh-ed25519
2021-07-23 21:21:45,561 paramiko.transport Cipher agreed: aes128-ctr
2021-07-23 21:21:45,561 paramiko.transport MAC agreed: hmac-sha2-256
2021-07-23 21:21:45,562 paramiko.transport Compression agreed: none
2021-07-23 21:21:45,598 paramiko.transport kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2021-07-23 21:21:45,599 paramiko.transport Switch to new keys ...
2021-07-23 21:21:45,607 p=1 u=root |  ec2-18-231-152-229.sa-east-1.compute.amazonaws.com | UNREACHABLE! => {
    "changed": false,
    "msg": "[Errno 2] No such file or directory: 'Web_SP.pem'",
    "unreachable": true
}
2021-07-23 21:21:48,089 p=1 u=root |  ERROR! the playbook: docker_ubuntu.yml could not be found
2021-07-23 21:36:38,875 p=1 u=root |  ansible 2.8.4
  config file = /ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.7.5 (default, Oct 17 2019, 12:25:15) [GCC 8.3.0]
2021-07-23 21:36:38,875 p=1 u=root |  Using /ansible/ansible.cfg as config file
2021-07-23 21:36:38,916 p=1 u=root |  host_list declined parsing /ansible/hosts as it did not pass it's verify_file() method
2021-07-23 21:36:38,916 p=1 u=root |  script declined parsing /ansible/hosts as it did not pass it's verify_file() method
2021-07-23 21:36:38,917 p=1 u=root |  auto declined parsing /ansible/hosts as it did not pass it's verify_file() method
2021-07-23 21:36:38,921 p=1 u=root |  Parsed /ansible/hosts inventory source with ini plugin
2021-07-23 21:36:39,599 p=1 u=root |  META: ran handlers
2021-07-23 21:36:39,681 p=1 u=root |  <ec2-15-228-50-128.sa-east-1.compute.amazonaws.com> ESTABLISH PARAMIKO SSH CONNECTION FOR USER: ubuntu on PORT 22 TO ec2-15-228-50-128.sa-east-1.compute.amazonaws.com
2021-07-23 21:36:39,908 paramiko.transport starting thread (client mode): 0x6cb22a90
2021-07-23 21:36:39,909 paramiko.transport Local version/idstring: SSH-2.0-paramiko_2.4.2
2021-07-23 21:36:39,943 paramiko.transport Remote version/idstring: SSH-2.0-OpenSSH_7.6p1 Ubuntu-4ubuntu0.3
2021-07-23 21:36:39,943 paramiko.transport Connected (version 2.0, client OpenSSH_7.6p1)
2021-07-23 21:36:39,964 paramiko.transport kex algos:['curve25519-sha256', 'curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group16-sha512', 'diffie-hellman-group18-sha512', 'diffie-hellman-group14-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2021-07-23 21:36:39,965 paramiko.transport Kex agreed: ecdh-sha2-nistp256
2021-07-23 21:36:39,965 paramiko.transport HostKey agreed: ssh-ed25519
2021-07-23 21:36:39,966 paramiko.transport Cipher agreed: aes128-ctr
2021-07-23 21:36:39,966 paramiko.transport MAC agreed: hmac-sha2-256
2021-07-23 21:36:39,966 paramiko.transport Compression agreed: none
2021-07-23 21:36:39,997 paramiko.transport kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2021-07-23 21:36:39,999 paramiko.transport Switch to new keys ...
2021-07-23 21:36:40,008 p=1 u=root |  ec2-15-228-50-128.sa-east-1.compute.amazonaws.com | UNREACHABLE! => {
    "changed": false,
    "msg": "[Errno 2] No such file or directory: 'Web_SP.pem'",
    "unreachable": true
}
2021-07-23 19:23:33,324 p=8031 u=root |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost
does not match 'all'

2021-07-23 19:23:33,601 p=8031 u=root |   ____________
< PLAY [all] >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 19:23:33,602 p=8031 u=root |  skipping: no hosts matched
2021-07-23 19:23:33,621 p=8031 u=root |   ____________
< PLAY RECAP >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 19:23:54,254 p=8096 u=root |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost
does not match 'all'

2021-07-23 19:23:54,428 p=8096 u=root |   ____________
< PLAY [all] >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 19:23:54,429 p=8096 u=root |  skipping: no hosts matched
2021-07-23 19:23:54,450 p=8096 u=root |   ____________
< PLAY RECAP >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 19:24:05,837 p=8129 u=root |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost
does not match 'all'

2021-07-23 19:24:05,845 p=8129 u=root |  ERROR! playbooks must be a list of plays

The error appears to have been in '/opt/git/capacitacaodevops/Terraform_Aws_Ansible_Docker_Jenkins/Ansible/hosts': line 1, column 1, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


ec2-15-228-127-63.sa-east-1.compute.amazonaws.com ansible_python_interpreter=/usr/bin/python3 ansible_ssh_private_key_file=keys/Web_SP.pem ansible_user=ubuntu
^ here

2021-07-23 19:24:23,230 p=8175 u=root |  ERROR! the playbook: ../Ansible/ does not appear to be a file
2021-07-23 19:33:04,656 p=10135 u=root |   ____________
< PLAY [all] >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 19:33:04,689 p=10135 u=root |   ________________________
< TASK [Gathering Facts] >
 ------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 19:33:15,150 p=10135 u=root |  fatal: [ec2-15-228-50-128.sa-east-1.compute.amazonaws.com]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host ec2-15-228-50-128.sa-east-1.compute.amazonaws.com port 22: Connection timed out\r\n", "unreachable": true}
2021-07-23 19:33:15,154 p=10135 u=root |  	to retry, use: --limit @/opt/git/capacitacaodevops/Terraform_Aws_Ansible_Docker_Jenkins/Ansible/docker_ubuntu.retry

2021-07-23 19:33:15,188 p=10135 u=root |   ____________
< PLAY RECAP >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 19:33:15,188 p=10135 u=root |  ec2-15-228-50-128.sa-east-1.compute.amazonaws.com : ok=0    changed=0    unreachable=1    failed=0   
2021-07-23 20:11:20,604 p=17405 u=root |  ERROR! the playbook: ../Ansible does not appear to be a file
2021-07-23 20:11:24,308 p=17421 u=root |  ERROR! the playbook: ../Ansible does not appear to be a file
2021-07-23 20:11:33,768 p=17446 u=root |   [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost
does not match 'all'

2021-07-23 20:11:33,943 p=17446 u=root |   ____________
< PLAY [all] >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 20:11:33,944 p=17446 u=root |  skipping: no hosts matched
2021-07-23 20:11:33,964 p=17446 u=root |   ____________
< PLAY RECAP >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 20:11:41,412 p=17465 u=root |   ____________
< PLAY [all] >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 20:11:41,446 p=17465 u=root |   ________________________
< TASK [Gathering Facts] >
 ------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 20:11:51,970 p=17465 u=root |  fatal: [ec2-15-228-50-128.sa-east-1.compute.amazonaws.com]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host ec2-15-228-50-128.sa-east-1.compute.amazonaws.com port 22: Connection timed out\r\n", "unreachable": true}
2021-07-23 20:11:51,972 p=17465 u=root |  	to retry, use: --limit @/opt/git/capacitacaodevops/Terraform_Aws_Ansible_Docker_Jenkins/Ansible/docker_ubuntu.retry

2021-07-23 20:11:52,001 p=17465 u=root |   ____________
< PLAY RECAP >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 20:11:52,002 p=17465 u=root |  ec2-15-228-50-128.sa-east-1.compute.amazonaws.com : ok=0    changed=0    unreachable=1    failed=0   
2021-07-23 20:14:02,632 p=18628 u=root |   ____________
< PLAY [all] >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 20:14:02,667 p=18628 u=root |   ________________________
< TASK [Gathering Facts] >
 ------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 20:14:12,939 p=18628 u=root |  fatal: [ec2-15-228-50-128.sa-east-1.compute.amazonaws.com]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host ec2-15-228-50-128.sa-east-1.compute.amazonaws.com port 22: Connection timed out\r\n", "unreachable": true}
2021-07-23 20:14:12,943 p=18628 u=root |  	to retry, use: --limit @/opt/git/capacitacaodevops/Terraform_Aws_Ansible_Docker_Jenkins/Ansible/docker_ubuntu.retry

2021-07-23 20:14:12,990 p=18628 u=root |   ____________
< PLAY RECAP >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||


2021-07-23 20:14:12,991 p=18628 u=root |  ec2-15-228-50-128.sa-east-1.compute.amazonaws.com : ok=0    changed=0    unreachable=1    failed=0   
